### PR TITLE
Made some minor changes to GTN FAQs

### PR DESCRIPTION
**Pull request by Melkeb**
- Changed Import an History to Importing History
- Removed the space to delete the unnecessary name info seen on the image
Capture

### CODE REVIEW

File: faqs/galaxy/collections_rename.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ area: collections
 box_type: tip
 layout: faq
 contributors: [delphine-l, martenson, shiltemann, hexylena]
-
```
Comment (shiltemann) [inline faqs/galaxy/collections_rename.md:7]: huh, this empty line was causing the bad rendering? nice catch! We should probably add a check for that to the github testing framework.

File: faqs/galaxy/histories_import.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Import an history
+title: Importing history
```
Comment (shiltemann) [inline faqs/galaxy/histories_import.md:2]: I would make it either "Importing histori**es**" (plural), or "Importing **a** history"

Comment (Melkeb) [reply faqs/galaxy/histories_import.md:2]: Yeah to make it consistent with the other questions on the GTN history section I will use the second one. Thanks

File: faqs/galaxy/collections_rename.md (hunk 1 above)
Comment (Melkeb) [reply faqs/galaxy/collections_rename.md:7]: This should be tested though if it actually makes a difference and removes the unnecessary info. I didn't see anything wrong with it besides this inconsistency compared to the other files.

Comment (shiltemann) [reply faqs/galaxy/collections_rename.md:7]: I just built the website locally and it is indeed fixed, so nice find :+1:

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks @Melkeb! :tada:

**Pull request merged**